src/main/scala/io/sqooba/conf/Transformer.java

```diff
@@ -0,0 +1,5 @@
+package io.sqooba.conf;
+
+public interface Transformer<T> {
```

Comment:
Maybe add a little comment on top of this class in case people land in it by walking through code?

Reply:
ok